src/main/java/org/prebid/server/bidder/model/CompositeBidderResponse.java

```diff
@@ -0,0 +1,40 @@
+package org.prebid.server.bidder.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.proto.openrtb.ext.response.FledgeAuctionConfig;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Composite bidder response (bids + other data) returned by a {@link Bidder}.
+ */
+@AllArgsConstructor(staticName = "of")
+@Builder(toBuilder = true)
+@Value
```

Comment:
Can be simplified
```
@Builder(toBuilder = true)
@Value(staticConstructor = "of")
```

Reply:
This results in a compile error for this line in Bidder.java
`CompositeBidderResponse.of(result.getValue(), result.getErrors(), null)`:

> src/main/java/org/prebid/server/bidder/Bidder.java:45:42
java: cannot find symbol
  symbol:   method of(java.util.List<org.prebid.server.bidder.model.BidderBid>,java.util.List<org.prebid.server.bidder.model.BidderError>,<nulltype>)
  location: class org.prebid.server.bidder.model.CompositeBidderResponse

because the constructor provided by `@Value` is overridden by the one from `@Builder`.

Reply:
Ok. Then leave it with
```
@Value
@Builder(toBuilder = true)
```
And, please, replace constructor invocations with builder.

```
@Value
@Builder(toBuilder = true)
public class CompositeBidderResponse {

    @Builder.Default
    List<BidderBid> bids = Collections.emptyList();

    @Builder.Default
    List<BidderError> errors = Collections.emptyList();

    /**
     * FLEDGE interest group bids passback
     */
    List<FledgeAuctionConfig> fledgeAuctionConfigs;

    public static CompositeBidderResponse empty() {
        return builder().build();
    }

    public static CompositeBidderResponse withBids(List<BidderBid> bids,
                                                   List<FledgeAuctionConfig> fledgeAuctionConfigs) {

        return builder()
                .bids(bids)
                .fledgeAuctionConfigs(fledgeAuctionConfigs)
                .build();
    }

    public static CompositeBidderResponse withError(BidderError error) {
        return builder().errors(Collections.singletonList(error)).build();
    }
}
```